src/collections/_documentation/sdks/javascript/errors/context.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Event Context
+excerpt: ""
+---
+
+{%- capture errors_configure_tags -%}
+Most SDKs generally support configuring tags by configuring the scope:
+
+```js
+Sentry.setTag("page_locale", "de-at");
+```
+
+{%- endcapture -%}
+
+{%- capture errors_release_name -%}
+
+For JavaScript developers, a release is also used for applying source maps to minified JavaScript to view original, untransformed source code.
+
+{%- endcapture -%}
+
+{%- capture __errors_set_level -%}
+
+To set the level out of scope, you can call `captureMessage()` per event:
+
+```js
+Sentry.captureMessage('this is a debug message', 'debug');
+```
+
+To set the level within scope, you can call `setLevel()`:
+
+```js
+Sentry.configureScope(function(scope) { 
+   scope.setLevel(Sentry.Severity.Warning);
+});
+```
+
+or per event:
+
+```js
+Sentry.withScope(function(scope) { 
+   scope.setLevel("info"); Sentry.captureException(error);
```

Comment:
```suggestion
   scope.setLevel("info");
   Sentry.captureException(error);
```

Reply:
Also not sure if we want to use `Sentry.Severity.Info` here or just `"info"`. Both works just fine.

Reply:
I will change it to `Sentry.captureException("info")`